doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -172,6 +172,12 @@ To support the FMDN extension, set the **Find My Device** feature to **true** fo
 
 For an example that uses the **Find My Device** feature, see the :ref:`fast_pair_locator_tag` sample.
 
+.. note::
+   To test the FMDN extension with the debug (uncertified) device models, you must set up your Android test device.
+   Make sure your phone uses the primary email account that is registered on Google's email allow list for the FMDN feature.
+   To register your development email account, complete Google's device proposal form.
+   The link to the form is available in the blue banner note at the beginning of `Fast Pair Find My Device Network extension` specification.
```

Comment:
```suggestion
   The link to the device proposal form is available at the beginning of the `Fast Pair Find My Device Network extension`_ specification.
```
I'm not entirely sure if we should even reference it's at the beginning, since we don't maintain this doc, and have no control whether it's moved somewhere or not. 
Here's simplified suggestion:

```suggestion
   You can find the link to the device proposal form in the `Fast Pair Find My Device Network extension`_ specification.
```

Reply:
Sure, this information is also phrased in such a general manner in our marketing material:

https://www.nordicsemi.com/Products/Technologies/Find-my-Device (Get started developing section)